src/hooks/useConfirmationText.ts

```diff
@@ -18,19 +18,12 @@ function useConfirmationText() {
   useEffect(() => {
     if (location.state && location.state.confirmationText) {
       setConfirmationText(location.state.confirmationText);
-      history.replace({
```

Comment:
Do we need a new version of this so we don't affect current implementations by removing this code?

Reply:
To my understanding,  we were clearing out confirmation text because the `Accessibility` (top level component w/ routes) view never un-mounts. That's where the confirmation text was.

Now that the confirmation text lives each individual route/view, the text is cleared when un-mounting.

I don't think this affects the GRT/GRB/Business Owner flow because we don't have confirmation text. They only display on the home page.